e2e/test/scenarios/filters-reproductions/dashboard-filters-reproductions.cy.spec.js

```diff
@@ -205,12 +205,12 @@ describe("issue 8030 + 32444", () => {
         H.saveDashboard();
 
         cy.wait("@getCardQuery");
-        cy.get("@getCardQuery.all").should("have.length", 4);
```

Comment:
This repro spec seems very suspicious. It should be verifying that we do not reload queries for dashcards that weren't connected to parameters.
- Initially we load dashboard with two cards and `@getCardQuery.all` should be 2 on line 190
- Then we connect only one card to the parameter as the second one gets connected and immediately disconnected
- As we should not reload the query for the second dashcard `@getCardQuery.all` must be 3 instead of 4. It is 4 only if we reload queries for both cards